packages/fuselage/src/components/Badge/styles.scss

```diff
@@ -0,0 +1,82 @@
+@use '../../styles/colors.scss';
+@use '../../styles/lengths.scss';
+@use '../../styles/typography.scss';
+
+$badge-colors-secondary-color: theme('badge-colors-secondary-color', colors.foreground(primary));
+$badge-colors-secondary-background-color: theme('badge-colors-secondary-background-color', colors.primary(100));
+
+$badge-colors-primary-color: theme('badge-colors-primary-color', colors.foreground(alternative));
+$badge-colors-primary-background-color: theme('badge-colors-primary-background-color', colors.primary(500));
+
+$badge-colors-danger-color: theme('badge-colors-danger-color', colors.foreground(alternative));
+$badge-colors-danger-background-color: theme('badge-colors-danger-background-color', colors.danger(500));
+
+$badge-colors-ghost-color: theme('badge-colors-ghost-color', colors.foreground(alternative));
+$badge-colors-ghost-background-color: theme('badge-colors-ghost-background-color', colors.neutral(700));
+
+$badge-colors-warning-color: theme('badge-colors-warning-color', colors.foreground(alternative));
+$badge-colors-warning-background-color: theme('badge-colors-warning-background-color', colors.warning-alternative(500));
+
+$badge-colors-disabled-color: theme('badge-colors-disabled-color', colors.foreground(hint));
+$badge-colors-disabled-background-color: theme('badge-colors-disabled-background-color', colors.neutral(400));
+
+.rcx-badge {
+  display: flex;
+  overflow: hidden;
+
+  width: fit-content;
+  min-width: lengths.size(16);
+
+  padding:
+    lengths.padding(2)
+    lengths.padding(4);
+
+  white-space: nowrap;
+
+  text-decoration: none;
+  text-overflow: ellipsis;
+
+  word-break: keep-all;
+
+  border-radius: lengths.border-radius(4);
+
+  @include typography.use-font-scale(micro);
+
+  &--secondary {
+    color: $badge-colors-secondary-color;
+    background-color: $badge-colors-secondary-background-color;
+  }
+
+  &--primary {
+    color: $badge-colors-primary-color;
+    background-color: $badge-colors-primary-background-color;
+  }
+
+  &--danger {
+    color: $badge-colors-danger-color;
+    background-color: $badge-colors-danger-background-color;
+  }
+
+  &--warning {
+    color: $badge-colors-warning-color;
+    background-color: $badge-colors-warning-background-color;
+  }
+
+  &--ghost {
+    color: $badge-colors-ghost-color;
+    background-color: $badge-colors-ghost-background-color;
+  }
+
+  &--disabled {
+    color: $badge-colors-disabled-color;
+    background-color: $badge-colors-disabled-background-color;
+  }
+
+  &--round {
```

Comment:
I don't think this is necessary anymore. You can apply this style directly to the component. I had used the `rounded` prop because I was using the same component for tags and badges. But badges are always round.